lib/brain.js

```diff
@@ -1546,6 +1546,96 @@ Brain.prototype.twoWayMode = function twoWayMode () {
   return this.trader.twoWayMode
 }
 
+/**
+ * Check if the customer is suspended
+ *
+ * That happens if the customer has reached
+ * one of the enabled compliance tier thresholds
+ *
+ * @name isSuspended
+ * @function
+ *
+ * @param {object} customer Acting customer
+ * @param {date} now Current date
+ * @returns {bool} Whether customer is suspended or not
+ */
+Brain.prototype.isSuspended = function isSuspended (customer, now) {
+  return customer && customer.suspendedUntil && new Date(customer.suspendedUntil) > now
+}
+
+/**
+ * Display the suspended screens for customer
+ * If the customer hasn't inserted bills yet,
+ * the suspendedCustomer screen will displayed with ok button,
+ * else the bill screen will be displayed with the relative error message
+ *
+ * @name showSuspendedCustomer
+ * @function
+ *
+ * @param {object} customer Acting customer
+ * @param {date} now Current date
+ *
+ */
+Brain.prototype.showSuspendedCustomer = function showSuspendedCustomer (customer, now) {
+  const data = this.hardLimitReached(customer, now)
+
+  /*
+   * When doing cashOut just show the hardLimitReached screen
+   */
+  if (this.tx.direction === 'cashOut') {
+    return this._timedState('hardLimitReached', { data })
+  }
+
+  /*
+   * Current transaction's fiat not including current bill
+   */
+
+  const insertedBills = this.tx.fiat.gt(0)
+  if (!insertedBills) {
+    return this._timedState('hardLimitReached', { data })
+  }
+
+  /*
+   * Set acceptingBills first as transition (in updateBillScreen) so that sendOnly
+   * reason message would be displayed on that screen
+   */
+  this.updateBillScreen(true)
+    .then(() => {
+      this.browser().send({
+        sendOnly: true,
+        reason: 'blockedCustomer',
+        cryptoCode: this.tx.cryptoCode
+      })
+    })
+}
+
+/*
+* Calculates the time difference between the
+* current date and the suspension time
+*
+* @name hardLimitReached
+* @function
+*
+* @param {object} customer Acting customer
+* @param {date} now Current date
+* @returns {object} hard limit
+*/
+Brain.prototype.hardLimitReached = function hardLimitReached (customer, now) {
```

Comment:
Can we rename this? By the name looks like we're testing for a boolean. 

Maybe `getHardLimitReachedData`